tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -38,15 +38,23 @@ sub run {
 
     #	bootloader options and set probe foreign OS, timeout
     assert_and_click 'yast2-bootloader_bootloader-options';
+
+    wait_still_screen 10;
+    send_key 'alt-t';    # Import untrusted GnuPG key -- workaround
+    wait_still_screen 10;
+
     assert_screen 'yast2-bootloader_bootloader-options-switched';
     send_key 'alt-b';
     wait_still_screen 3;
     send_key 'alt-t';
     type_string '16';
 
     #	default boot section
+    for (1 .. 2) { send_key 'alt-f10' }
     assert_and_click 'yast2-bootloader_default-boot-section';
```

Comment:
we could do a `check_screen($tag)` to see if it is visible this area and if not run the workaround.
right now we are clicking before make it visible due to the workaround, it would make sense to only click after, right?

Reply:
"it would make sense to only click after, right?"---sorry, I cannot get it. I added the  `for` loop because the window is not fully loaded, this caused the next line `assert_and_click` failed

Reply:
I guess we are not planning to do the workaround in a better way just to apply it, never mind :)